docs/user_guide/tensorflow/hops.rst

```diff
@@ -145,28 +157,28 @@ The ``experiment`` module is used for running one or more Parallel TensorFlow ex
     # job2: lr=0.1 and dropout=0.7
     # job3: lr=0.3 and dropout=0.4
     # job4: lr=0.3 and dropout=0.7
-    
+
     from hops import experiment
 
     root_tensorboard_logdir = experiment.grid_search(spark, grid_experiments_wrapper, args_dict, direction='max')
-    
+
     ...............................................................................................
-    
+
     def evolutionary_experiments_wrapper(lr, dropout):
         # Wrapper function for arbitrarily many experiments
         metric = mycode.evaluate(lr, dropout)
         return metric
-        
+
     # Running a grid of hyperparameter experiments
     search_dict = {'lr': [0.1, 0.3], 'dropout': [0.4, 0.7]}
-    
+
     from hops import experiment
     root_tensorboard_logdir = experiment.evolutionary_search(spark, evolutionary_experiments_wrapper, search_dict, direction='max')
-    
-    
+
+
 tensorboard
 ------------------------------
-Hops supports TensorBoard for all TensorFlow modes (Parallel experiments, TensorFlowOnSpark and Horovod). 
+Hops supports TensorBoard for all TensorFlow modes (Parallel experiments, TensorFlowOnSpark and Horovod).
```

Comment:
@Limmen @robzor92 Haven't we ditched Horovod yet?

Reply:
Yep, still some places to update

Reply:
@Limmen Can you remove it here?